services/QuillLMS/client/app/bundles/Teacher/components/general_components/tooltip/activity_details.jsx

```diff
@@ -1,54 +1,58 @@
 import React from 'react';
 import moment from 'moment';
-import activityTypeFromClassificationId from '../../modules/activity_type_from_classification_id.js';
 
-export default class ActivityDetails extends React.Component {
-  getClassName = () => {
-    if (this.props.data.concept_results && this.props.data.concept_results.length) {
+export const ActivityDetails = ({ data }) => {
+
+  if (!Object.keys(data).length) { return <span /> }
+
+  const { concept_results, started_at, updated, scores, activity_description } = data;
```

Comment:
@emilia-friedberg @anathomical that's correct, the reason it's `updated` is because of how the data was organized here in the Scorebook component: https://github.com/empirical-org/Empirical-Core/blob/develop/services/QuillLMS/client/app/bundles/Teacher/containers/Scorebook.jsx#L209

so I suppose I could change that to `updated_at` if y'all would prefer that?

Reply:
Oh, I see. I would just go with what's already there - though I hate it. It would be nice to clean up some of these files at some point.